model/src/test/kotlin/licenses/ResolvedLicenseTest.kt

```diff
@@ -50,8 +50,7 @@ class ResolvedLicenseTest : StringSpec({
 
         resolvedCopyrights shouldHaveSize 1
         with(resolvedCopyrights.first()) {
-            // TODO: The "(C)" should come directly after "Copyright".
-            statement shouldBe "Copyright 2019, 2022 (C) The ORT Project Authors"
+            statement shouldBe "Copyright (C) 2019, 2022 The ORT Project Authors"
```

Comment:
Should this commit auto resolve the Github issue ?

Reply:
No, the issue is actually about another problem that I *tried* to reproduce with this test, but instead discovered this problem.